.env.example

```diff
@@ -2,6 +2,7 @@ PGDATABASE=db
 PGPASSWORD=password
 PGUSER=postgres
 PGHOST=localhost
+CLIENT_DOMAIN=http://io-onboarding:8080
```

Comment:
why io-onboarding? Is it ok to indicate just _localhost_?

Reply:
we want to limit here just the fe application which is currently on the same domain

Reply:
It's ok, this is just an example file in order to show what the required environment variables are: the real values should be given according to the actual environment wherein the app is running.